ppb/__init__.py

```diff
@@ -1,8 +1,10 @@
 from ppb.vector import Vector
 from ppb.engine import GameEngine
 
-from ppb.scenes import BaseScene
 from ppb.sprites import BaseSprite
+# Needs to come after BaseSprite because it technically depends on it.
+from ppb.scenes import BaseScene
```

Comment:
I would have thought that module resolution would have handled this? But I'm guessing this is here because you had a bug.

Reply:
Short version: Circular import problems. Should maybe set a policy to not use the `__init__.py` exports in internal modules and to use the modules directly.

Reply:
That's a good idea.